### ISSUE TITLE
Is it possible to have performance metrics for your store?

### ISSUE THREAD

**Issue by legomushroom** [zalmoxisus/redux-devtools-extension#223]
Hi!

I've installed the extension with the hope that it can give me insight into performance of my app. Particularly, I'm interested in how long it takes for all reducers to derive new state and maybe find out where is the bottleneck could be. Do you have any plans to add that?

Thanks!

**Comment by zalmoxisus** [comment]
Thanks for the suggestion! Yes, I started to implement this [here](https://github.com/zalmoxisus/redux-devtools-instrument/pull/4), but stopped due to the lack of interest on this feature. Here's how it would work: https://youtu.be/cbXLohVbzNI?t=373

**Comment by legomushroom** [comment]
Nice! Are you still planning to release your work on that matter?

**Comment by zalmoxisus** [comment]
Yes, I have a busy roadmap for the next version, then will try to implement that.
